src/System.Data.Common/tests/System/Data/Common/RowUpdatedEventArgsTest.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Xunit;
+
+namespace System.Data.Common
+{
+    public class RowUpdatedEventArgsTest
+    {
+        [Fact]
+        public void ConstructorFail()
+        {
+            try
+            {
+                new RowUpdatedEventArgs(null, null, (StatementType)666, null);
+                Assert.False(true);
+            }
+            catch (ArgumentOutOfRangeException ex)
```

Comment:
You can use `ArgumentOutOfRangeException e = Assert.Throws<ArgumentOutOfRangeException>(() => new RowUpdatedEventArgs(null, null, (StatementType)666, null));`

Reply:
Good to know, thanks 👍 
Will use that instead then